kube-dns-controller-aws-route53/kube-dns-controller-aws-route53-deployment.yaml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
Can you add a comment with a link to the big this is a workaround for

Reply:
done